lib/cmds/simulator.js

```diff
@@ -26,9 +27,9 @@ class Simulator {
     let port = (options.port || this.blockchainConfig.rpcPort || 8545);
 
     cmds.push("-p " + (port + (useProxy ? constants.blockchain.servicePortOnProxy : 0)));
-    if (!ganache) {
-      cmds.push("-h " + host);
-    }
+    // if (!ganache) {
+    //   cmds.push("-h " + host);
+    // }
```

Comment:
We still need the host. You can remove the if though.

Reply:
My interp of the old code:  that `if` only comes into play if `testrpc` was not falsy but `ganache` was falsy, whereas if both had been falsy then the script would have already bailed with `process.exit()` (see disabled code several lines above the `if` in question).

`ganache` will never be falsy with the new code — it's always assigned the value `path.join(...)`. Moreover, we can rely on that being okay since `ganache-cli` is a dependency of embark and `<embark>/node_modules/.bin/ganache-cli` will always be on the fs after embark is installed.

Reply:
ganache-cli does support a `-h` flag:

https://github.com/trufflesuite/ganache-cli/blob/develop/cli.js#L67

So maybe we weren't passing it to `ganache-cli` in the past, but it's okay/good to do so? Will experiment.

Reply:
Passing the `-h` flag to ganache-cli works fine, change made as you suggested.